sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -446,4 +451,75 @@ export class InstitutionController extends BaseController {
       },
     }));
   }
+
+  /**
+   * Get the Institution details for the ministry institution detail page
+   * @param institutionId
+   * @returns AESTInstitutionDetailDto
+   */
+  @AllowAuthorizedParty(AuthorizedParties.aest)
+  @Groups(UserGroups.AESTUser)
+  @Get("/:institutionId/detail")
+  async getAESTInstitutionDetailById(
+    @Param("institutionId") institutionId: number,
+  ): Promise<AESTInstitutionDetailDto> {
+    const institutionDetail =
+      await this.institutionService.getAESTInstitutionDetailById(institutionId);
+    return {
+      legalOperatingName: institutionDetail.legalOperatingName,
+      operatingName: institutionDetail.operatingName,
+      primaryPhone: institutionDetail.primaryPhone,
+      primaryEmail: institutionDetail.primaryEmail,
+      website: institutionDetail.website,
+      regulatingBody: institutionDetail.regulatingBody,
+      institutionTypeName: institutionDetail.institutionType.name,
+      formattedEstablishedDate: getExtendedDateFormat(
```

Comment:
We have been doing this here and there. In general formatting the dates to be displayed really seems to be a concern from the UI, not from the API.
I would recommend formatting to the UI only for exceptions where there is a motivation.
For now, thanks for creating the method on the date-utils.